spec/cocina/models/validators/language_tag_validator_spec.rb

```diff
@@ -0,0 +1,117 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe Cocina::Models::Validators::LanguageTagValidator do
+  let(:validate) { described_class.validate(clazz, props) }
+
+  let(:props) { file_props }
+
+  let(:file_props) do
+    {
+      externalIdentifier: 'bc123df4567_1',
+      label: 'Page 1',
+      type: Cocina::Models::ObjectType.file,
+      version: 1,
+      access: { view: 'dark', download: 'none' },
+      administrative: {
+        publish: false,
+        shelve: false,
+        sdrPreserve: true
+      },
+      hasMessageDigests: [],
+      hasMimeType: 'text/plain',
+      filename: 'page1.txt'
+    }
+  end
```

Comment:
basically lifted from `dark_validator_spec.rb` to start, and collapsed some of its `let` vars that wouldn't change in this spec into the base properties.  didn't try to simplify more than that, but happy to do so if that's possible/preferable.